### ISSUE TITLE
[Feature] - Please add support for Enchanted Shulkers

### ISSUE THREAD

**Issue by DemonSlayer112** [Aton-Kish/reinforced-shulker-boxes#51]
Description:
I been trying to tell which of the two shulker mods i like more, yours or "Upgraded Shulkers"
Yours easily won, in funcationality with another mod already and texture wise.
But a mod called [Item Swapper](https://www.curseforge.com/minecraft/mc-mods/itemswapper) adds a functionality to drag items from shulkers.
Sadly the mod doesnt work with this mod, but it would be great if it could! :)

What is wished to happen:
For compatibility with the Item Swapper mod.

### Additional context

Item Swapper mod link: https://www.curseforge.com/minecraft/mc-mods/itemswapper

**Comment by Aton-Kish** [comment]
I have found there was also the same issue in the Item Swapper mod repository. https://github.com/tr7zw/ItemSwapper/issues/117
Let me think about it.

**Comment by DemonSlayer112** [comment]
Forgot to mention that i been using the 1.19.2 version, if that would be an issue, ik most mods have been pushing for 1.19.3 for some reason.

**Comment by Aton-Kish** [comment]
I'm pretty sure there were some breaking changes between 1.19.2 and 1.19.3, but I don't think that matters. The Item Swapper mod doesn't seem to have an interface for integrating with other mods. So, I'll have to figure out how to proceed.

**Comment by DemonSlayer112** [comment]
should we also ask on their github for if they can help open for interfacing?
Btw ty for the time for looking into it :D

**Comment by Aton-Kish** [comment]
Yeah. Specifically, having the [entrypoint](https://fabricmc.net/wiki/documentation:entrypoint) of the Item Swapper mod would make the integration easier.

**Comment by allanonmage** [comment]
I found a compatibility mod that does something similar to what we need:
https://modrinth.com/mod/charm-drag-and-drop-shulker-fix

```
This is a very simple mod to improve the behavior of Charm's drag and drop shulker module.

The module's normal behavior with any shulker, be it modded or not, is to use the vanilla shulker box's container size, even if a modded shulker, in this case a shulker from Reinforced Shulkers, has a larger size than a vanilla shulker box.

This mod patches that to make it so that, at least a Reinforced Shulker, can be used to it's full capacity with the module.
```


This or something like this might improve the compatibility between your mod and Enchanced Shulkers.  The dev seems friendly too.
